helpers/__init__.py

```diff
@@ -34,15 +34,38 @@ def wait_for_grafana_url_generation(visualizer_url, job_id, max_wait_time=120, m
                 return url
     raise Exception("Max tries to get generated URL exceeded.")
 
+def wait_for_job_complete(manager_url, job_id, max_wait_time=120, max_tries=10):
+    """ Waits for a defined time for a job to be completed.
+            Arguments:
+                manager_url {string} -- The URL from the Asperathos Manager
+                job_id {string} -- Job id generated when the job is submitted
+                max_wait_time {int} -- Maximum time, in seconds, to be waited
+                max_tries {int} -- Maximum times this function will try to get the URL
+            Returns:
+                url {string} -- Grafana URL
+            Raises:
+                Exception -- When max_tries is reached without success
+    """
+    delta = max_wait_time / max_tries
+    for _ in range(max_tries):
+        sleep(delta)
+        response = requests.get(manager_url + '/submissions/{}'.format(job_id))
+        if response.ok:
+            status = response.json().get('status')
+            if status == "completed" and response.json().get('final_replicas'):
+                    return True
+
+    raise Exception("Max tries to job get completed.")
```

Comment:
I don't get this Exception. The message should be something like "Max retries for job to be completed exceeded." ?